app/views/search/search.html.erb

```diff
@@ -50,6 +50,12 @@
                 <%= link_to dataset.title, dataset_path(dataset.uuid, dataset.name) %>
               </h2>
               <dl class="dgu-metadata__box">
+                <% if !dataset.released? %>
```

Comment:
I'm not sure this predicate method exists? Maybe we could change the implementation in `app/models/dataset.rb` so the revert commit applied cleanly and we don't have to change `app/views/datasets/_meta_data.html.erb`?

```rb
def released?
  released.present?
end
```